asv_bench/benchmarks/io/csv.py

```diff
@@ -98,6 +99,44 @@ def time_frame_date_no_format_index(self):
         self.data.to_csv(self.fname)
 
 
+class ToCSVPeriod(BaseIO):
+    fname = "__test__.csv"
+
+    params = ([1000, 10000], ["D", "H"])
+    param_names = ["obs", "fq"]
```

Comment:
can these be "nobs" and "freq", would match patterns elsewhere

Reply:
Ok. I just originally recopied what was already there in `ToCSVDatetimeBig`. Changing all.